installer/msi/wasmtime.wxs

```diff
@@ -67,13 +67,17 @@
             <Component Id="wasmtime.exe" Guid="*">
                 <File Id="wasmtime.exe" Source="target\release\wasmtime.exe" KeyPath="yes" Checksum="yes"/>
             </Component>
+            <Component Id="wasmtime_utils.exe" Guid="*">
```

Comment:
The output filename is based on `Source` attribute ([docs](https://wixtoolset.org/documentation/manual/v3/xsd/wix/file.html), search for 'Name'). Previous CI build failed, because WiX IDs can't contain `-`.